### PR TITLE
Update archive-linkedin-data.md

### PR DESCRIPTION
**Pull request by msbemba**
added a line to Step 7 about the mailbox license requirement

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9579

File: microsoft-365/compliance/archive-linkedin-data.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ After the LinkedIn Company page data is stored in a mailbox, you can apply Micro
 
 6. Select the company page that you want to archive items from, and then click **Next**.
 
-7. On the **Choose storage location** page, click in the box, select the email address of a Microsoft 365 mailbox that the LinkedIn items will be imported to, and then click **Next**. Items are imported to the inbox folder in this mailbox.
+7. On the **Choose storage location** page, click in the box, select the email address of a Microsoft 365 mailbox that the LinkedIn items will be imported to, and then click **Next**. Items are imported to the inbox folder in this mailbox. The Mailbox used must be licensed with  exchange online plan 1 or 2.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/archive-linkedin-data.md]:
```suggestion
7. On the **Choose storage location** page, click in the box, select the email address of a Microsoft 365 mailbox that the LinkedIn items will be imported to, and then click **Next**. Items are imported to the inbox folder in this mailbox. The mailbox used must have an Exchange Online Plan 1 or Plan 2 license.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@v-tophillips Please merge this PR. Thanks

**Review by v-tophillips — APPROVED** [review]

**Pull request merged**